tox.ini

```diff
@@ -4,10 +4,12 @@ envlist = py36,py37,py38,flake8
 
 [testenv]
 commands = python -m unittest discover
+extras =
+    sql
 
 
 [testenv:flake8]
-deps = -r requirements/test.txt
+deps = -rrequirements/test.txt
```

Comment:
is this supposed to be `-r requirements/test.txt`?

Reply:
No, as including a space confuses the parsing on older tox versions:
```
ERROR: Could not open requirements file: [Errno 2] No such file or directory: ' requirements/test.txt'
```